### PR TITLE
Add notes on the PKI cert generation forwarding regression

### PR DESCRIPTION
**Pull request by sgmiller**

### CODE REVIEW

File: website/content/partials/pki-forwarding-bug.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## PKI Certificate Generation Forwarding Regression
+
+A bug introduced in Vault 1.8 causes certificates issue by the PKI secrets engine made on a performance
```
Comment (ncabatoff) [inline website/content/partials/pki-forwarding-bug.mdx:3]:
```suggestion
A bug introduced in Vault 1.8 causes certificates generation requests via the PKI secrets engine made on a performance
```

File: website/content/partials/pki-forwarding-bug.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+secondary node to be forwarded to a Vault cluster's primary node.  This affects horizontal scalability, but
```
Comment (ncabatoff) [inline website/content/partials/pki-forwarding-bug.mdx:4]: I don't expect the horizontal scalability is the real issue most users, it's more about the certs/CRLs being in the wrong place.  If we do want to retain this bit, I might expand it a little - not everyone's going to follow what we mean by horizontal scalability.

File: website/content/partials/pki-forwarding-bug.mdx (hunk 1 above)
Comment (mladlow) [reply website/content/partials/pki-forwarding-bug.mdx:3]: I think "causes certificate generation requests via" (get rid of the pluralization of "certificates")?

File: website/content/partials/pki-forwarding-bug.mdx (hunk 2 above)
Comment (sgmiller) [reply website/content/partials/pki-forwarding-bug.mdx:4]: Okay, I'll just drop that part.

**Comment by mladlow** [comment]
@sgmiller I changed the label to "backport/website" which will open and automerge a PR to stable website and open a PR against release/1.9 for versioned docs - could you please make sure to merge the release/1.9 PR? Those don't automerge.

**Review by mladlow — APPROVED** [review]
By my read this resolves @ncabatoff's comments but I don't necessarily want to speak for him. Anyway, I didn't understand this issue before reading this note and I now understand it much better, so from my perspective this is good.

**Pull request merged**